code/index.html

```diff
@@ -1,30 +1,110 @@
 <!DOCTYPE html>
 <html>
-  <head>
+
+<head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
-
-    <!-- Signup form -->
-
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
-
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
-
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
-    </section>
-    <!-- dont forget to link the javascript file here! -->
-  </body>
-</html>
+    <title>KREATISA</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header class="hero-image">
+        <div class="text-container">
+            <h1>KREATISA</h1>
+            <h2>Painting and pottery</h2>
+            <button class="class-btn"><a href="#requestForm">Schedule course</a></button>
+        </div>
+    </header>
+
+    <main class="wrapper">
+        <section class="form-section">
+            
+            <form action="https://httpbin.org/anything" method="POST">
+                <div class="request-form" id="requestForm">
+                    <h2>Request</h2>
```

Comment:
Perhaps this `<h2>`-tag and the one on line 54 could be `<h3>`-tags instead, but it's just a minor semantic detail.